src/blocks/ui/insights/personalization.js

```diff
@@ -21,15 +22,27 @@ const Personalization = ( {
 	block,
 	clientId,
 } ) => {
-	const [ days, setDays ] = useState( 7 );
+	const [ days, setDays ] = useState( 'P7D' );
+	const [ start, setStart ] = useState( 'P7D' );
+	const [ end, setEnd ] = useState( 'P7D' );
 	const analytics = useSelect( select => {
-		return select( 'analytics/xbs' ).getViews( clientId, { days } );
-	}, [ clientId, days ] );
+		return select( 'analytics/xbs' ).getViews( clientId, {
+			days,
+			start,
+			end,
+		} );
+	}, [ clientId, days, start ] );
 	const lift = useSelect( select => {
-		const current = select( 'analytics/xbs' ).getViews( clientId, { days: 7 } );
+		const current = select( 'analytics/xbs' ).getViews( clientId, {
+			days: 'P7D',
+			start: 'P7D',
+			end: 'P7D',
```

Comment:
I'm a bit confused by this, the start and end values should be date strings rather than durations. What's your aim with adding these here in this way rather than deriving a start and end from the duration? We shouldn't have both days, and start/end values as they're representing the same thing. This is pretty confusing as-is.